Libraries/Image/Image.android.js

```diff
@@ -186,6 +186,8 @@ const BaseImage = (props: ImagePropsType, forwardedRef) => {
       ? loadingIndicatorSource.uri
       : null,
     ref: forwardedRef,
+    accessible: props.alt !== undefined ? true : props.accessible,
+    accessibilityLabel: props.alt ?? props.accessibilityLabel,
```

Comment:
Should we swap the order?
```suggestion
    accessibilityLabel: props.accessibilityLabel ?? props.alt,
```

Reply:
I'm not sure, @necolas states here (https://github.com/facebook/react-native/issues/34424) that 
> Each new prop should take priority over any existing equivalents.

Should we take a different approach just with this component?

Reply:
I think the recommendation on web is that alt is required for contextual images and preferred over aria-label. So if we can only pick one or the other for native, alt should take priority

Reply:
Hmm I found mozilla site (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) says otherwise:

> If you give your [<iframe>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe)s a title, your images an alt attributes, and your input's associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)s, aria-label is not necessary. But, if present, the aria-label will take precedence over the title, alt and <label> as your iframe, image, or input's accessible name, respectively.

Reply:
Oh good catch. I suppose alt is preferred but label takes priority if provided.

Reply:
Should we swap the order then or keep it as is?

Reply:
Yes. `aria-label` should set `accessibilityLabel`, even if `alt` is provided.

Reply:
Fixed